components/network/openssh/files/ssh-askpass

```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/ksh
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright 2009 Sun Microsystems, Inc.  All rights reserved.
+# Use is subject to license terms.
+#
+
+prompt=$(echo $1 | sed s/_/__/g)
+ICON=/usr/share/pixmaps/blueprint-keyring.png
+exec /usr/bin/zenity --entry --title "ssh-askpass" \
+    --text="$prompt" --hide-text --window-icon=$ICON
```

Comment:
This is a ditto copy from older illumos-gate - what's wrong with formatting
here? Make it single-line?

Note even this will take uncertain time until I get to a computer with
editor and git and internet all at once, and something that anyone could
fix in another PR ;)

--
Typos courtesy of GMail on my Samsung Android

27.08.2016 7:57 пользователь "Alexander Pyhalov" <notifications@github.com>
написал:

> In components/network/openssh/files/ssh-askpass
> <https://github.com/OpenIndiana/oi-userland/pull/2365#discussion_r76511090>
> :
>
> > +# If applicable, add the following below this CDDL HEADER, with the
> > +# fields enclosed by brackets "[]" replaced with your own identifying
> > +# information: Portions Copyright [yyyy] [name of copyright owner]
> > +#
> > +# CDDL HEADER END
> > +#
> > +
> > +#
> > +# Copyright 2009 Sun Microsystems, Inc.  All rights reserved.
> > +# Use is subject to license terms.
> > +#
> > +
> > +prompt=$(echo $1 | sed s/_/__/g)
> > +ICON=/usr/share/pixmaps/blueprint-keyring.png
> > +exec /usr/bin/zenity --entry --title "ssh-askpass" \
> > +    --text="$prompt" --hide-text --window-icon=$ICON
>
> At least fix formatting
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenIndiana/oi-userland/pull/2365/files/32ad0332ec68da1cc613113981f2831980d44889#r76511090>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABj5lA2iLDa9UW1t07Yp4mjN1J3yuzvbks5qj9HagaJpZM4JtprB>
> .
>